src/calc/calculate_phaseZeroBalance.py

```diff
@@ -29,11 +29,9 @@ def calculate(self, reward_logs, reactivate_zeroed):
 
             # KT1 accounts do not require reactivation on 0 balance
             if (
-                delegate.type == reward_log.TYPE_DELEGATOR
```

Comment:
because the redirection can mean that the target address is not of TYPE_DELEGATOR? What is this type used for? Why do you think this has been here in the first place as a condition?

Reply:
The question of the hour is why this was here in the first place. This issue arrised for me when I added a founder with no balance.